src/ctia/http/routes/common.clj

```diff
@@ -134,3 +134,19 @@
     true {:refresh "wait_for"}
     false {:refresh "false"}
     {}))
+
+(s/defn capabilities->description :- s/Str
+  "Does not add leading or trailing new lines."
+  [capabilities :- (s/conditional
+                     keyword? (s/pred simple-keyword?)
+                     nil? (s/pred nil?)
+                     :else #{(s/pred simple-keyword?)})]
+  (cond
+    (keyword? capabilities) (str "Requires capability " (name capabilities) ".")
+    ((every-pred set? seq) capabilities) (-> (apply str "Requires capabilities "
+                                                    (->> capabilities
+                                                         sort
+                                                         (map name)
+                                                         (str/join ", ")))
+                                             (str "."))
+    :else "No capabilities needed."))
```

Comment:
We should ask QA to check this last case is never reached :-)

Reply:
I think it's reachable. Should we make it an error?

Reply:
@gbuisson wdyt?

Reply:
It looks like the `"/external_id/:external_id"` routes might not always have a capability. I'll do more digging.

Reply:
https://github.com/threatgrid/iroh/issues/4443

Reply:
Made an error.